src/common/buttons/button.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import styles from './button.module.scss';
+
+const DefaultProps = {
+  onClick: undefined,
+  isDisabled: undefined,
+};
+
+interface ButtonProps {
+  children: ReactNode;
+  onClick?: () => void;
+  isDisabled?: boolean;
+}
+
+const Button = (props : ButtonProps) => (
+  <button
+    className={styles.customButton}
+    type="submit"
```

Comment:
This button will be useful in other instances outside of forms. I think you could pass the `type` as a prop so it's not always set to `submit`.

Typescript wise you could do a 
```
type HTMLButton = ComponentPropsWithoutRef<'button'>;

interface ButtonProps {
// ...
type?: Pick<HTMLButton, 'type'>
}
```
and it could have `button` as default value